pom.xml

```diff
@@ -178,6 +178,7 @@
               <excludes>
                 <exclude>**/target/**</exclude>
                 <exclude>docs/**</exclude>
+                <exclude>src/ci/docker/**</exclude>
```

Comment:
let's add the copyright+license to each file instead.

Reply:
@michaelsembwever done